barretenberg/cpp/src/barretenberg/vm/avm/recursion/avm_recursive_verifier.test.cpp

```diff
@@ -0,0 +1,123 @@
+#include "barretenberg/vm/avm/recursion/avm_recursive_verifier.hpp"
+#include "barretenberg/circuit_checker/circuit_checker.hpp"
+#include "barretenberg/numeric/random/engine.hpp"
+#include "barretenberg/stdlib_circuit_builders/ultra_flavor.hpp"
+#include "barretenberg/sumcheck/instance/prover_instance.hpp"
+#include "barretenberg/ultra_honk/ultra_prover.hpp"
+#include "barretenberg/ultra_honk/ultra_verifier.hpp"
+#include "barretenberg/vm/avm/generated/circuit_builder.hpp"
+#include "barretenberg/vm/avm/generated/composer.hpp"
+#include "barretenberg/vm/avm/recursion/avm_recursive_flavor.hpp"
+#include "barretenberg/vm/avm/tests/helpers.test.hpp"
+#include "barretenberg/vm/avm/trace/common.hpp"
+#include "barretenberg/vm/avm/trace/helper.hpp"
+#include "barretenberg/vm/avm/trace/trace.hpp"
+#include <gtest/gtest.h>
+
+namespace tests_avm {
+
+using namespace bb;
+using namespace bb::avm_trace;
+
+namespace {
+auto& engine = bb::numeric::get_debug_randomness();
+}
```

Comment:
What is this for?

Reply:
This is usually used to get some "weak pseudo randomness" to have deterministic tests. In this prcise case, this is a leftover which we do not use. I will remove this.